.vscode/settings.json

```diff
@@ -25,6 +25,6 @@
     },
     "python.formatting.provider": "yapf",
     "[json]": {
-	    "editor.formatOnSave": false
+        "editor.formatOnSave": false
```

Comment:
Could this be reverted, as well?

Reply:
Or at least done in a separate PR, as the change itself is good (fixing mixed tabs and spaces).

Reply:
Updated